projects/core/src/i18n/translation.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { ServerConfig } from '../config/server-config/server-config';
+import { I18NextService } from './i18next/i18next.service';
+
+@Injectable()
+export class TranslationService {
+  private readonly NON_BREAKING_SPACE = String.fromCharCode(160);
+
+  constructor(
+    private i18NextService: I18NextService,
+    private config: ServerConfig
+  ) {}
+
+  exists(key: string, options: any = {}): boolean {
+    return this.i18NextService.exists(key, options);
+  }
+
+  translate(key: string, options: any = {}): string {
```

Comment:
Should we use this one by default, or rather `translateLazy`?

Reply:
Removed synchronous translate as it's unpredictable. And renamed translateLazy to translate.